app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/picture/PictureFragmentOfflineTest.kt

```diff
@@ -45,43 +46,55 @@ class PictureFragmentOfflineTest {
         // Initialize Firebase Storage
         storage = Firebase.storage
         storage.useEmulator("10.0.2.2", 9198)
-        emptyFirebaseStorage(storage)
-
-        // Empty Local Files
         emptyLocalFiles(picturesDir)
 
-        // Launch the fragment
+        // Open the drawer
         onView(ViewMatchers.withId(R.id.drawer_layout))
             .perform(DrawerActions.open())
-        onView(ViewMatchers.withId(R.id.pictureFragment))
+        onView(ViewMatchers.withId(R.id.roadBookFragment))
```

Comment:
You could import ViewMatchers methods individually so that you don't have to specify `ViewMatchers` every time